### ISSUE TITLE
Postbank BestSign Login

### ISSUE THREAD

**Issue by michaelgreyskull** [raphaelm/python-fints#89]
Hello,

thank you for your great software! :)
Since Postbank has established new ways of authentication, I wanted to ask if someone already managed to do a login with python-fints using the _BestSign_ authentication mechanism.

When you do the usual initializiation like `f = FinTS3PinTanClient(BLZ, PostbankID, password, 'https://hbci.postbank.de/banking/hbci.do')` and afterwards call `f.get_sepa_accounts()` there is this (new) error:

> Dialog response: 9050 - Teilweise fehlerhaft.
> Dialog response: 9999 - Dialog bereits geschlossen.
> Dialog response: 9800 - Dialog ungueltig/unbekannt.

Usually a login attempt triggers the BestSign app to authorize the login but in this case, I only get the error messages.
Has anyone connected succesfully using this BestSign method, yet?

Thanks to you!

**Comment by raphaelm** [comment]
> Has anyone connected succesfully using this BestSign method, yet?

I don't think so. I don't know enough about BestSign to decide if this is a duplicate to #72, though

**Comment by michaelgreyskull** [comment]
Thank you for your fast response!

It's not exactly a duplicate (that's why I opened up this issue specifically for BestSign) but definitely related as this method has been introduced due to PSD2. I hope they do not shut down the FinTS interface completely. I will contact Postbank and keep you updated.

**Comment by raphaelm** [comment]
I think this no longer applies, as I use the library with BestSign regularly
